### PR TITLE
Added new red themes and alternate version of all themes.

### PR DESCRIPTION
**Pull request by Inverted-E**
Added new red themes and alternate versions of all themes. These alternative themes change the QTableWidget hover behavior to highlight the whole row instead of individual cells, similar to how a file explorer works. See photos in README to see the difference between standard and alt themes.

All new themes were built and are available in resources.

### CODE REVIEW

**Comment by Inverted-E** [comment]
See photos in README to see the difference between standard and alt themes

**Comment by Inverted-E** [comment]
Just noticed you added some extension specific colors (as below) to the theme JSON files. These will need adding to my new themes before merging. Will do it tomorrow.

```json
    "ads-tab:focused": "rgba(xx, xx, xx, xx)",
    "ads-border:focused": "rgba(xx, xx, xxx, xx)"
```

File: theme/dark-green-alt.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#31363b",
+    "highlight": "#33b833",
+    "highlight:dark": "#2b992b",
+    "highlight:alternate": "#1f991f",
+    "midtone": "#76797c",
+    "midtone:light": "#b0b0b0",
+    "midtone:dark": "#626568",
+    "midtone:hover": "#8a8d8f",
+    "view:checked": "#325c32",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/dark-green-alt.json:16]: This should be `"transparent"` and not `"None"`. This is replaced via string replacement directly, so the correct styling would be a passthrough or transparent.

File: theme/dark-purple-alt.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#31363b",
+    "highlight": "#a74bb8",
+    "highlight:dark": "#872a99",
+    "highlight:alternate": "#851f99",
+    "midtone": "#76797c",
+    "midtone:light": "#b0b0b0",
+    "midtone:dark": "#626568",
+    "midtone:hover": "#8a8d8f",
+    "view:checked": "#57335e",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/dark-purple-alt.json:16]: Same as the other alts.

File: theme/light-blue-alt.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#eaebec",
+    "highlight": "rgba(51, 164, 223, 0.5)",
+    "highlight:dark": "rgba(45, 147, 200, 0.5)",
+    "highlight:alternate": "rgba(71, 184, 243, 0.6)",
+    "midtone": "#bab9b8",
+    "midtone:light": "#bab9b8",
+    "midtone:dark": "rgba(106, 105, 105, 0.7)",
+    "midtone:hover": "#787876",
+    "view:checked": "#b9dae7",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/light-blue-alt.json:16]: Same as the other alts.

File: theme/light-green-alt.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#eaebec",
+    "highlight": "rgba(50, 223, 50, 0.5)",
+    "highlight:dark": "rgba(45, 200, 45, 0.5)",
+    "highlight:alternate": "rgba(71, 243, 71, 0.6)",
+    "midtone": "#bab9b8",
+    "midtone:light": "#bab9b8",
+    "midtone:dark": "rgba(106, 105, 105, 0.7)",
+    "midtone:hover": "#787876",
+    "view:checked": "#b9e7b9",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/light-green-alt.json:16]: Same as the other alts.

File: theme/light-purple-alt.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#eaebec",
+    "highlight": "rgba(164, 51, 223, 0.5)",
+    "highlight:dark": "rgba(147, 45, 200, 0.5)",
+    "highlight:alternate": "rgba(184, 71, 243, 0.6)",
+    "midtone": "#bab9b8",
+    "midtone:light": "#bab9b8",
+    "midtone:dark": "rgba(106, 105, 105, 0.7)",
+    "midtone:hover": "#787876",
+    "view:checked": "#dab9e7",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/light-purple-alt.json:16]: Same as the other alts.

File: theme/light-red-alt.json [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    "background:alternate": "#eaebec",
+    "highlight": "rgba(223, 52, 52, 0.5)",
+    "highlight:dark": "rgba(200, 45, 45, 0.5)",
+    "highlight:alternate": "rgba(243, 73, 73, 0.6)",
+    "midtone": "#bab9b8",
+    "midtone:light": "#bab9b8",
+    "midtone:dark": "rgba(106, 105, 105, 0.7)",
+    "midtone:hover": "#787876",
+    "view:checked": "#da7c7c",
+    "view:hover": "None",
```
Comment (Alexhuszagh) [inline theme/light-red-alt.json:16]: Same as the other alts.

**Review by Alexhuszagh — CHANGES_REQUESTED** [review]
I'd also like to keep `light` and `dark`, since otherwise it could break existing code using those styles. What we could do is always register them as an alias in the compiled files, so loading from `light-blue` and `light` would do the same.

Otherwise, this looks great.

**Comment by Alexhuszagh** [comment]
What we could do is make it so we add an entry inside the `qrc` file as:

`<file>light-blue/stylesheet.qss</file>`

And this is just just a copy of `light/stylesheet.qss`. This way, we just add an alias and nothing else needs to change: no new resources, etc. I can add this on if you revert back to `light` and `dark` if that works?

**Comment by Alexhuszagh** [comment]
#103 has been merged, so you can rebase off of that, override with the new resource files, etc., and change these lines from `light` and `dark` to `light-blue` and `dark-blue`.

https://github.com/Alexhuszagh/BreezeStyleSheets/blob/12bb954c2fc135c4a5f3a362b6b8e8d29c6f4884/configure.py#L36

https://github.com/Alexhuszagh/BreezeStyleSheets/blob/12bb954c2fc135c4a5f3a362b6b8e8d29c6f4884/configure.py#L417

This line then should have the inverse (our aliases): `light-blue` and `dark-blue` to `light` and `dark`.

https://github.com/Alexhuszagh/BreezeStyleSheets/blob/12bb954c2fc135c4a5f3a362b6b8e8d29c6f4884/configure.py#L338

**Pull request closed without merging**